config/schedule.rb

```diff
@@ -18,3 +18,8 @@
 # end
 
 # Learn more: http://github.com/javan/whenever
+
+# TODO: Raise this to monthly eventually?
+every :sunday do
+  runner 'DownloadJob.enqueue_all'
```

Comment:
Could we comment this out for now? Or what until we expand our storage and account for that?